### DISCUSSION TITLE
Recognizing Pyenv kernels

### DISCUSSION THREAD

**Discussion by BillCartwright** [posit-dev/positron#3711] [Q&A]
Hello.  I'm having trouble getting Positron to recognize my Pyenv kernels and environments.  I have tried adding `~/.pyenv/ ` to Python Path in settings but the kernels still do not show up.

Interestingly, in the Positron terminal I do see my pyenv environments, and can launch a terminal instance of a Pyenv Python environment.

**Comment by seeM** [comment]
Could you please share more about your environments? For example, what is the output of `pyenv versions` in the terminal?

How are you trying to see the kernels available in Positron? Have you tried the "Python: Select Interpreter" command?

Note that if discovery fails, as a last resort you can always manually add an interpreter via "Python: Select Interpreter" > "Enter interpreter path...".
